ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/_layout/runs/$runIndex/vulnerabilities/index.tsx

```diff
@@ -252,18 +252,15 @@ const VulnerabilitiesComponent = () => {
       <CardHeader>
         <CardTitle>Vulnerabilities (ORT run global ID: {ortRun.id})</CardTitle>
         <CardDescription>
-          <p>
-            These are the vulnerabilities found currently in the project. Please
-            note that the vulnerability status may change over time, as your
-            project dependencies change. Therefore, your project should be
-            scanned for vulnerabilities regularly.
-          </p>
+          These are the vulnerabilities found currently in the project. Please
+          note that the vulnerability status may change over time, as your
+          project dependencies change. Therefore, your project should be scanned
+          for vulnerabilities regularly.
           <br />
-          <p>
-            By clicking on "References" you can see more information about the
-            vulnerability. The overall severity rating is calculated based on
-            the highest severity rating found in the references.
-          </p>
+          <br />
+          By clicking on "References" you can see more information about the
+          vulnerability. The overall severity rating is calculated based on the
+          highest severity rating found in the references.
         </CardDescription>
```

Comment:
It makes sense to have multiple `p`s here, so looking at `CardDescription` (it's a simple `p`), I think the best solution would just to use two of them like:

```typescript
<CardDescription>
  These are the vulnerabilities...
</CardDescription>
<CardDescription>
  By clicking on...
</CardDescription>
```

Reply:
Oh, didn't know you can actually have several...